### PR TITLE
feat:flipkart mobile view completed

### PR DESCRIPTION
**Pull request by Amaljose616**

### CODE REVIEW

File: flipkart/style.css [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+    height: 24px;
+}
+.sort-icon img, .filter-icon img{
+    width: 100%;
+    height: 100%;
+}
+.sort-txt, .filter-txt{
+    font-size: 14px;
+    font-family:Roboto Medium,Roboto-Medium,Droid Sans,HelveticaNeue-Medium,Helvetica Neue Medium,sans-serif-medium;
+    transform: translateX(-0.5px);
```
Comment (vonnue-akhil) [inline flipkart/style.css]: why translate? Try if there is any other way

File: flipkart/style.css [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+    transform: translateY(-1px);
+}
+.element-logo img{
+    width: 32px;
+    height: 32px;
+}
+.element-txt-special,.element-txt{
+    font-size: 12px;
+    color: #212121;
+    font-family:myfont;
```
Comment (vonnue-akhil) [inline flipkart/style.css]: name the fonts with their name instead of myfont

**Review by vonnue-akhil — CHANGES_REQUESTED** [review]
Where is screen shots

File: flipkart/style.css (hunk 1 above)
Comment (Amaljose616) [reply flipkart/style.css]: i think it's the simple way to make it pixel perfect

File: flipkart/style.css (hunk 2 above)
Comment (Amaljose616) [reply flipkart/style.css]: ok

**Comment by Amaljose616** [comment]
127 0 0 1_5500_index html

**Pull request merged**